sampledb/reader.xsh

```diff
@@ -0,0 +1,92 @@
+import pandas as pd
+from time import time
+from sampledb.sampledatabase import SampleDatabase
+
+
+def get_uid_from_qr():
+    a = $(zbarcam)
+    b = a.split()
+    c = [z.strip('QR-Code:') for z in b]
+    d = set(c)
```

Comment:
We should use an OrderedDict here so that we preserve the order of the unique elements.